### PR TITLE
Add a faq for contributor survey

### PR DESCRIPTION
**Pull request by yaahc**
also includes some minor changes from when I was entering the survey into surveyhero as a separate commit.

Note, I feel like I got a little too informal when filling out the FAQ entries for the last two questions, so I'd appreciate review and feedback on those sections in particular.

### CODE REVIEW

File: surveys/2021-contributor-survey/FAQ.md [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+
+## How much time will it take to answer the survey?
+
+On average, it should take from 5 to 15 minutes.
+
+## What kind of questions are included in the survey?
+
+This survey gathers demographic information that can be used to group feedback
+based on areas of involvment and feedback questions that seek to identify
+specific positives and negatives within the rust project contribution
```
Comment (technetos) [inline surveys/2021-contributor-survey/FAQ.md:24]: Capitalize Rust

File: surveys/2021-contributor-survey/FAQ.md [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+experience.
+
+## How will we use the data from the survey responses?
+
+The answers from the survey will be anonymized, aggregated, and summarized. A
+high level writeup will be posted to blog.rust-lang.org.
```
Comment (technetos) [inline surveys/2021-contributor-survey/FAQ.md:30]: Might consider making this an actual link to blog.rust-lang.org

File: surveys/2021-contributor-survey/FAQ.md [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+Nearly every question in the survey is optional. You are welcome to share as
+much or as little information as you are comfortable with. Only the community
+survey team will have access to the raw data, and only anonymized aggregate
+data is shared with others.
+
+This year’s survey was commissioned by the Rust Foundation and your responses
+are subject to the Foundation’s [privacy
+policy](https://foundation.rust-lang.org/policies/privacy-policy).
+
+## Where and when is the survey results report published?
```
Comment (technetos) [inline surveys/2021-contributor-survey/FAQ.md:43]: s/is/are

File: surveys/2021-contributor-survey/FAQ.md [hunk 4]
```diff
@@ -0,0 +1,85 @@
...
+feel comfortable identifying yourself as such.
+
+The primary difference is that the follow up questions asked of active
+contributors assume a greater familiarity with the contribution process than
+those asked of potential contributors and non-contributors.
+
+## What is the difference between a team, working group, and a project group? How can I tell how my team/group is classified?
+
+Team, working group, and project group are classifications that the project
+uses to loosely classify the various groups that make up the rust project.
```
Comment (technetos) [inline surveys/2021-contributor-survey/FAQ.md:75]: s/rust/Rust

File: surveys/2021-contributor-survey/FAQ.md [hunk 5]
```diff
@@ -0,0 +1,85 @@
...
+These groups are identified in the [team
+repo](https://github.com/rust-lang/team), and can be disambiguated by the
+`kind` field in their toml entry in the `team/teams` folder.
+
+- team have no `kind` field (e.g. compiler team)
```
Comment (technetos) [inline surveys/2021-contributor-survey/FAQ.md:80]: s/team/teams

**Comment by apiraino** [comment]
With a bit of regret, I am going to close this because I think the content is so out sync with the current State of Rust annual survey that it's probably easier to cherry pick specific points and add them to the README.md (which I will take note to do)

**Pull request closed without merging**